javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,219 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  let newArray= [];
+  arr.forEach(value =>{
+    value++;
+    newArray.push(value);
+  });    return newArray;
```

Comment:
put your return on a new line. This will work but it's poorly formatted. Wouldn't fly in a professional environment.